src/main/resources/META-INF/resources/webjars/service/prov/prov.js

```diff
@@ -2620,7 +2620,7 @@ define(function () {
 		 * Usage text renderer.
 		 */
 		usageToText: function (usage) {
-			return usage ? usage.text || (usage.name + '<span class="pull-right">(' + usage.rate + '%)<span>') : null;
+			return usage ? usage.rate === 100 ? usage.name +'<small class="pull-right">'+usage.duration+' mois</small>': usage.text|| (usage.name +'<small class="pull-right">'+usage.duration+' mois <small>' + '<span class="pull-right"> (' + usage.rate + '%)<span>') : null;
```

Comment:
Can you use the format as defined in #172? It should not  is no such "mois"?
Also, the render "small" should integrate the whole right text, and not only the rate